ssc/ONBOARDING.md

```diff
@@ -0,0 +1,24 @@
+# SPIFFE Steering Committee Onboarding
+
+Congratulations!
```

Comment:
some suggestion:

## Introduction Meeting

Schedule an introductory meeting for new SSC members with current members. This can facilitate a smoother transition and allow new members to understand the committee’s ongoing initiatives and dynamics.

## Documentation Access

Explicitly list any key documents (besides the README.md) that new members need to read or have access to.

## Feedback Mechanism

Introduce a feedback mechanism for new members to share their onboarding experience and suggest improvements.

## Emergency Contact Information

Collect emergency contact information (and secure them safely, of course) of the members for any urgent communication needs.

## Exit Interview

Not sure if it makes sense; but could be nice to have a session where the parting member can share their thoughts and feeling, and suggestion on how to make SSC even better.

Reply:
added all except contact info; the SSC doesn't directly maintain things that could have urgent security reports, which is all I know of that maintainers get paged for.

do we have anything currently other than slack and regular meetings? is more presence there all that's needed? I am curious to know what sort of situations you have in mind where one SSC member must urgently contact another, but is not something that the one SSC member would be already empowered by our charter to handle.